### DISCUSSION TITLE
E-book list with first to last serial number/ division for beginners to learn as course.

### DISCUSSION THREAD

**Discussion by Arifulebn** [dat-linux/community#19] [App, Extra, Book suggestions]
I am very happy about DAT Linux OS. I am a beginner and want to gain knowledge about Data Science.

I send you a screenshot (attached) of the e-book list. I am new to this line. As a result, I can not understand which book I read first.  

In here, the books store by name, size, owner etc. 

Please can you give me a serial number of e-book list such as course code/ division. That's why I can start easily with no doubt/ confusion.
![screen.jpg](https://user-images.githubusercontent.com/121567527/210183501-346272e2-502a-4fa1-aa83-90b65c7dc957.jpg)

**Comment by dat-linux** [comment]
These books cover a range of topics in data science, there is no "serial number" or such for them that would make organisation into categories useful. Most books would fit into different categories. Feel free to browse the listings and bookmark as you think would suit your learning program. (The icons are shortcuts, and should open in a browser. Switching to "compact view" might help readability in the file manager).

For beginners to data science, I would recommend:

An Introduction to Data Analysis - Michael Franke (for R)
or
Learning Data Science - Lau, Gonzalez, & Nolan (for Python)
